devnet-sdk/system2/chain.go

```diff
@@ -0,0 +1,56 @@
+package system2
+
+import (
+	"github.com/ethereum/go-ethereum/params"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+// Chain is an ethereum chain, common between L1 and L2.
+// For L1 or L2 specifics, see L1Chain and L2Chain extensions.
+// A chain hosts configuration resources and tracks participating nodes.
+type Chain interface {
+	Common
+
+	ChainID() eth.ChainID
+
+	ChainConfig() *params.ChainConfig
```

Comment:
Shouldn't this be `ChainConfig` instead of `params.ChainConfig`?

Reply:
`ChainConfig` in this package is a bit confusing. It was named that way because it's the config to the default implementation of the test `Chain` interface. It's not the same as the chain protocol configuration.
I'll try to adjust the naming to not be as confusing.